tests/assertions.py

```diff
@@ -33,16 +33,15 @@ def assert_meta(envelope, release="test-example-release"):
         "transaction": "test-transaction",
         "tags": {"expected-tag": "some value"},
         "extra": {"extra stuff": "some value", "…unicode key…": "őá…–🤮🚀¿ 한글 테스트"},
-        "sdk": {
-            "name": "sentry.native",
-            "version": "0.4.2",
-            "packages": [
-                {"name": "github:getsentry/sentry-native", "version": "0.4.2"},
-            ],
-        },
+    }
+    expected_sdk = {
+        "name": "sentry.native",
+        "version": "0.4.2",
+        "packages": [{"name": "github:getsentry/sentry-native", "version": "0.4.2"},],
     }
 
     assert matches(event, expected)
+    assert matches(event["sdk"], expected_sdk)
```

Comment:
I don't really get the test.  You're not actually seeing the backend thing you just added but just re-wrote this a bit?

Reply:
I basically want to exclude that from the match, as the assertion fails if we have additional attributes. And I don’t really want to pass in that flag explicitly for each testcase.

Reply:
Is it possible to add a test that checks it's present but `in ["crashpad", "breakpad", "inproc"]`?

Reply:
its not present for "none" ;-)

Reply:
Can you then at least test it in a separate assertion?